### ISSUE TITLE
v-select noDataText tile is disabled by default after #982. It should be controlled via prop.

### ISSUE THREAD

**Issue by chancet1982** [vuetifyjs/vuetify#1304]
After fixing #982 it seems that now the "no data" tile is disabled by default. Before this fix this was not the case. This is caused by **generators.js, line 100** where:
`children.push(this.genTile(this.noDataText, true))
`
This prevents any action being taken if a user is searching for an item which is currently not found (for example if item is not found then noDataText can be "add new" and by listening to the input/change you could trigger an method to add this missing item.

I think a better approach here instead of assuming that the "no data" tile should be non-clickable is to provide a prop (noDataTileDisabled -type: Boolean, default: true) and use that instead of the hardcoded 'true'

Basically:
`children.push(this.genTile(this.noDataText, this.noDataTileDisabled ))
`
This will open the door to "add new" if searchValue isnt in the items

### Steps to reproduce
1. Add a v-select to your project with autocomplete
2. Start typing (searching for an item that is not in the list)
3. Click on the "no data found" tile.

### Versions
All versions. change happened upgrading from 0.14.4 to 0.14.8
 
### What is expected ?
No data found item shouldnt be disabled by default, instead it should be controlled via prop.

### What is actually happening ?
item is now disabled, i believe it should be controlled via prop

### Reproduction Link
https://jsfiddle.net/chancet1982/knp56m9w/

**Comment by chancet1982** [comment]
After fixing #982 it seems that now the "no data" tile is disabled by default. Before this fix this was not the case. This is caused by generators.js, line 100 where:
`children.push(this.genTile(this.noDataText, true))`

This prevents any action being taken if a user is searching for an item which is currently not found (for example if item is not found then noDataText can be "add new" and by listening to the input/change you could trigger an method to add this missing item.

I think a better approach here instead of assuming that the "no data" tile should be non-clickable is to provide a prop (noDataTileDisabled -type: Boolean, default: true) and use that instead of the hardcoded 'true'

Basically:
`children.push(this.genTile(this.noDataText, this.noDataTileDisabled ))`

This will open the door to "add new" if searchValue isnt in the items
